game/Makefile

```diff
@@ -1,7 +1,7 @@
 CXX = g++
 CXXFLAGS = -std=c++14 -Wall -MMD
 EXEC = hackmon
-OBJECTS = action.o
```

Comment:
Wow you removed action.o :(
![bitmoji](https://render.bitstrips.com/v2/cpanel/4933dbc0-b957-4029-b002-8f890fa20fda-29b03f31-567b-40a9-836c-bbce3d642229-v1.png?transparent=1&palette=1&width=246)